packages/server/routes/api/events.js

```diff
@@ -114,4 +171,84 @@ router.get(
   }
 );
 
+router.delete(
+  "/:eventId",
+  AuthController.authenticate,
+  eventIdValidator,
+  async (request, response) => {
+    const result = validationResult(request);
+
+    // if validation result is not empty, errors occurred
+    if (!result.isEmpty()) {
+      response.status(400).json({
+        status: "fail",
+        data: result.array(),
+      });
+      return;
+    }
+
+    try {
+      const { eventId } = request.params;
```

Comment:
Use `matchedData(request)` to access input fields from now on since we're using express-validator. See docs for that [here](https://express-validator.github.io/docs/guides/getting-started#accessing-validated-data).